packages/lesmis/src/views/HomeView.js

```diff
@@ -107,12 +114,16 @@ const Footer = styled(MuiContainer)`
   right: 0;
 `;
 
-const FooterInner = styled(FlexEnd)`
+const FooterInner = styled(FlexSpaceBetween)`
   color: white;
   border-top: 1px solid rgba(255, 255, 255, 0.1);
   padding-top: 1rem;
   padding-bottom: 1rem;
 
+  @media screen and (min-width: 600px) {
+    padding-left: 1.5rem;
+  }
```

Comment:
What's this bit doing?

Reply:
It's aligning the logos on desktop screens to line up with the info text. The layout could be shuffled around so that the padding is using the same number as the 50px in the main container above but since it's a one off layout I am happy to have this specific little bit in here for now.